charts/beta/vector/README.md

```diff
@@ -85,3 +85,51 @@ Make sure to update as appropriate for your use case.
 ```
 
 Note: make sure to replace the `<monitoring-namespace>` in the loki endpoint with the namespace where you installed loki.
+
+Sample override to ingest from cloudtrail:
```

Comment:
can we clarity that we are talking about AWS CloudTrail, and also provide a link to their service docs?